### PR TITLE
chore: longer default cordonDrainTimeout for Azure Stack Cloud

### PR DESCRIPTION
**Pull request by jiaxunsongucb**
**Reason for Change**:

**Issue Fixed**:

**Credit Where Due**:

Does this change contain code from or inspired by another project?

- [X] No
- [ ] Yes

If "Yes," did you notify that project's maintainers and provide attribution?

- [ ] No
- [ ] Yes

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:
Set 60 minutes cordonDrainTimeout for Azure Stack Cloud to give it enough time to move around resources during Node Drain, especially disk detach/attach operations.
We still honor the user's input.

### CODE REVIEW

File: cmd/upgrade.go [hunk 1]
```diff
@@ -170,6 +170,13 @@ func (uc *upgradeCmd) loadCluster() error {
 		return errors.Wrap(err, "error parsing the api model")
 	}
 
+	// Set 60 minutes cordonDrainTimeout for Azure Stack Cloud to give it enough time to move around resources during Node Drain,
```
Comment (jadarsie) [inline cmd/upgrade.go:173]: Maybe method `validate` is a better place for this change. Please add a unit test.

Comment (jiaxunsongucb) [reply cmd/upgrade.go:173]: I've tried to add this change in method validate, but the problem is that uc.containerService.Properties.IsAzureStackCloud() is not accessible in method validate, it must be after "uc.containerService, uc.apiVersion, err = apiloader.LoadContainerServiceFromFile(uc.apiModelPath, true, true, nil)" is called.

Comment (jadarsie) [reply cmd/upgrade.go:173]: right

**Pull request merged**